artifacts/534/deployment.json

```diff
@@ -0,0 +1,7 @@
+{
+	"gasPrice": 0,
```

Comment:
is gasprice 0 expected?

Reply:
Hey guys @mmv08 @Uxio0 Candle is a scam chain, I was personally scammed by the main "Ceo" who took 10 ETH from me, and also they scammed hundreds of other developers, who I spoke to and confirmed.  The guy who scammed everyone = Of course his name was Sam lol , his dad worked for ripple, so just a heads up, do not add candle!!

Reply:
We trust ethereum-lists as a source of trust for valid ethereum chains, and only enable chains to be able to deploy the singleton. We don't analyze or investigate the project as we don't provide support for it

Reply:
Ok awesome, thank you 🙏